ERROR	zm_deactivate_swoole (ERROR 503)

1. Tell us your PHP version(`php -v`)

7.1.7

2. Tell us your Swoole version(`php --ri swoole`)

2.2.0

3. Tell us your `Laravel`/`Lumen` version(check composer.json & composer.lock)

Lumen (5.4.7)

4. Detail description about this issue(error/log)

1、启动 php artisan laravels  start  -d
2、curl http://127.0.0.1:5200

报错：

[2018-08-28 18:38:24 *12547.4]	ERROR	zm_deactivate_swoole (ERROR 503): Fatal error: Uncaught Error: __clone method called on non-object in /data/home/bornli/wwwroot/ctp-core/vendor/hhxsv5/laravel-s/src/Illuminate/Laravel.php:326
Stack trace:
#0 /data/home/bornli/wwwroot/ctp-core/vendor/hhxsv5/laravel-s/src/LaravelS.php(121): Hhxsv5\LaravelS\Illuminate\Laravel->__clone()
#1 {main}
  thrown in /data/home/bornli/wwwroot/ctp-core/vendor/hhxsv5/laravel-s/src/Illuminate/Laravel.php on line 326.
[2018-08-28 18:38:24 $12513.0]	WARNING	swManager_check_exit_status: worker#4 abnormal exit, status=255, signal=0
[2018-08-28 18:38:24] [ERROR] LaravelS: worker[4] error: exitCode=255, signal=0

5. Give us a `reproducible` code block and `steps`

--- a/src/Illuminate/Laravel.php
+++ b/src/Illuminate/Laravel.php
@@ -323,6 +323,9 @@ class Laravel
     public function __clone()
     {
         $this->app = clone $this->app;
-        $this->laravelKernel = clone $this->laravelKernel;
+
+        if (!$this->conf['is_lumen']) {
+            $this->laravelKernel = clone $this->laravelKernel;
+        }
     }
 }

